src/lib/select/select.spec.ts

```diff
@@ -2360,9 +2380,9 @@ describe('MdSelect', () => {
   selector: 'basic-select',
   template: `
     <div [style.height.px]="heightAbove"></div>
-    <md-select placeholder="Food" [formControl]="control" [required]="isRequired"
-      [tabIndex]="tabIndexOverride" [aria-label]="ariaLabel" [aria-labelledby]="ariaLabelledby"
-      [panelClass]="panelClass">
+    <md-select placeholder="Food" [formControl]="control"
+      [required]="isRequired && !hideRequiredMarker" [tabIndex]="tabIndexOverride"
```

Comment:
again, switch to setting `hideRequiredMarker` instead of adding to `required` conditions